libvcx/src/settings.rs

```diff
@@ -73,7 +73,10 @@ pub static DEFAULT_WALLET_KEY: &str = "8dvfYSt5d1taSd6yJdpjq4emkwsPDDLYxkNFysFD2
 pub static DEFAULT_THREADPOOL_SIZE: usize = 8;
 pub static MASK_VALUE: &str = "********";
 pub static DEFAULT_WALLET_KEY_DERIVATION: &str = "RAW";
+#[cfg(not(target_os = "macos"))]
 pub static DEFAULT_PAYMENT_PLUGIN: &str = "libnullpay.so";
+#[cfg(target_os = "macos")]
+pub static DEFAULT_PAYMENT_PLUGIN: &str = "libnullpay.dylib";
```

Comment:
Good catch, is it related to the logging? Was it causing any issues?

Reply:
It's not related to logging actually - separate fix. Any tests trying to load libnullpay would fail on OSX because it was tried to load `.so`, while correctly it should be looking for `.dylib`